### DISCUSSION TITLE
Ctcreate runing error

### DISCUSSION THREAD

**Discussion by xiaozhh007** [nrc-cnrc/EGSnrc#1098] [Questions]
The program runs fine when I run the example, but when I open the dicom image sequence, which I exported using the Eclipse TPS, it doesn't work and reports the following error:
![1](https://github.com/nrc-cnrc/EGSnrc/assets/138585202/04164ae8-9285-4884-919e-e30b342455a5)
When I changed it to open only 1 .dcm file, reports the following error:
![2](https://github.com/nrc-cnrc/EGSnrc/assets/138585202/cc2d22a2-11d7-4572-abaa-968de8c91400)
And, I'm sure the dicom file get all data needed by ctcreate.
![3](https://github.com/nrc-cnrc/EGSnrc/assets/138585202/5fc09019-8c8f-43d6-80c4-0c68251cdbb1)

This error has been bothering me for days, any help with this would be greatly appreciated.

**Comment by rtownson** [comment]
The first file it's asking for is actually a text file, not a DICOM file. Inside the text file should be a list of all the filenames of the DICOM files, one filename per line.

**Reply by xiaozhh007** [reply]
Thank very much for the reply, but the first file contains what you said.
![4](https://github.com/nrc-cnrc/EGSnrc/assets/138585202/159e6229-98b9-415e-9d60-c1d838e5e2c8)
